app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -109,6 +153,19 @@ export const ListingDetailsFeatures = ({ listing, imageSrc }: ListingDetailsFeat
 
           <Description term={t("listings.features.unitFeatures")} description={""} />
         </dl>
+        {Object.keys(units).length > 0 ? (
```

Comment:
Sorry if this is easy to figure out and I just couldn't find an example of it while testing - is it possible that there'd be a valid API response with a length of 0 and the user would see an infinite spinner?

Reply:
oops, I had the same question. I'd think no, but I'll ask the team to be sure on Monday

Reply:
@kramduckner christine and I had a quick convo on [slack](https://sfdigitalservices.slack.com/archives/CCV0019TN/p1673290249038089) about this and I went with a quick solution that uses state to set a loading status. There's probably a more robust way to handle this, like lifting a loading state/spinner up into a higher-order component but that feels like scope creep. I'll add it as a nice-to-have ticket though!